packages/schema-generator/API.md

```diff
@@ -16,6 +16,10 @@ export type SchemaGeneratorConfig = {
         secretName: string;
         value: string;
     };
+    sslCert?: {

```

Comment:
```suggestion
    sslCertificate?: {

```

Reply:
Most CLI tools use the term `sslCert`.

**Reference:**
Postgres connection string: https://www.postgresql.org/docs/current/libpq-ssl.html#LIBPQ-SSL-CLIENTCERT
`netsh` commands: https://learn.microsoft.com/en-us/windows/win32/http/show-sslcert